src/main/java/edu/oakland/arttour/service/ArtTourService.java

```diff
@@ -0,0 +1,256 @@
+package edu.oakland.arttour.service;
+
+import edu.oakland.arttour.dao.ArtTourDAO;
+import edu.oakland.arttour.model.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ArtTourService {
+
+  @Autowired private ArtTourDAO dao;
+
+  public void addArtwork(Map<Object, Object> artwork) {
```

Comment:
Why do we do this?

Reply:
so we're adding an artwork and creator at the same time. I figured it would be some kinda form on the frontend, and we just pass all the information together.

Since some of the data are strings, and some are doubles, ints, etc. I had to use Object. I'm realizing now that it could be `Map<String, Object>` tho

Reply:
Ohh gotcha.
So from the frontend we could pass an object similar to the Artwor, that's why I've created interfaces to match the exact types of the models. Do you think passing the model here would work too?

Reply:
It could probably work. Im kinda rusty on using models in the controller

Reply:
Okay worth looking into it later